doc/release-notes-bucash.md

```diff
@@ -36,68 +37,567 @@ In case you decide to downgrade from BUcash 1.2.0.1, or greater, to a version ol
 will need to run the old release using `-reindex` option so that the
 UTXO will be rebuild using the previous format. Mind you that downgrading to version
 lower than 1.1.2.0 you will be split from the rest of the network that are following
-the rules activated Nov 13th 2017 protocol upgrade.
+the rules activated Nov 13th 2017 and May 15th 2018 protocol upgrades.
 
 Main Changes
 ------------
 
-- provide test infrastructure to create transactions with more complex script
-- various fix and improvements applied to BU request manager
-- fix socket select bug on windows platform
-- fix a problem with incoming connections due to node IP address discoverability
-- Ensure that EB and datacarrier size are set properly after a node restart
-- fix spurious disconnect due to block download crowding out getheaders
-- Prune slow peers during initial sync
+- Graphene Relay: A protocol for efficiently relaying blocks across a blockchain's network (experimental)
+- blocksdb: Add leveldb as an alternative storage method for blocks and undo data
+- Double Spend Relaying
+- BIP 135: Generalized version bits voting
+- clean up shadowing/thread clang warn
+- Update depends libraries
+- Rework of the Bitcoin fuzzer command line driver tool
+- Add stand alone cpu miner to the set of binaries
```

Comment:
add "This cpu miner showcases the new mining RPC calls, provides a template for development of mining pool software, and is valuable for regtest/testnet mining.  CPU mining on mainnet with this or any other software is an exercise in futility"

Reply:
done